src/helpers/generateModel.test.ts

```diff
@@ -83,18 +83,24 @@ test("it generates a model!", () => {
           isUnique: false,
         },
       ],
+      schema: null,
       primaryKey: null,
+      dbName: null,
       uniqueFields: [],
       uniqueIndexes: [],
-      dbName: null,
     },
     {
       databaseProvider: "sqlite",
       fileName: "",
       enumFileName: "",
       camelCase: false,
       readOnlyIds: false,
-    }
+      groupBySchema: false,
+      defaultSchema: "public",
+      dbTypeName: "DB",
+    },
+    false,
+    "public"
```

Comment:
Nitpick: generateModel should/could accept an options object instead here. The new arguments here are hard to immediately understand when scanning usages of this function.
```suggestion
      groupBySchema: false,
      defaultSchema: "public",
      dbTypeName: "DB",
    },
    false,
    "public"
```